packages/ssi-types/__tests__/dif_pe_examples/vp/vp_did_comm.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
since we don't support/use this, remove it

Reply:
removed and dir name changed